ironfish/src/account/accounts.ts

```diff
@@ -630,10 +642,21 @@ export class Accounts {
         }
 
         if (!map.spent) {
+          let confirmedMinimumBlocks = false
+
+          const { blockHash } = transactionMapValue
```

Comment:
nit: do we need to create more variables in this scope, can we just use `transactionMapValue.blockHash`?

Reply:
Agreed, I am not a fan of `transactionMapValue` either. I refactored the loop to use `blockHash` and `transaction` directly [here](https://github.com/iron-fish/ironfish/pull/1571/files#diff-975265306b3138d4067a44c1f74a98ef80249217a05ac91dff96236b482c429bR631).